### PR TITLE
nginx metrics

### PR DESCRIPTION
**Pull request by calellowitz**
First commit addresses https://dimagi-dev.atlassian.net/browse/SAAS-12588
Second commit addresses https://dimagi-dev.atlassian.net/browse/SAAS-12589

### CODE REVIEW

File: roles/logstash/templates/10-filters.conf.j2 [hunk 1]
```diff
@@ -129,6 +136,31 @@ filter {
...
+                mutate {
+                    add_field => { "duration" => "lt_020s" }
+                }
+            } else if [request_time] < 120 {
+                mutate {
+                    add_field => { "duration" => "lt_120s" }
+                }                
+            } else {
+                mutate {
+                    add_field => { "duration" => "over_120s" }
```
Comment (snopoke) [inline roles/logstash/templates/10-filters.conf.j2:159]: I think you also need to add the duration to the output config

Comment (calellowitz) [reply roles/logstash/templates/10-filters.conf.j2:159]: ah, i see. added as a tag to the nginx requests output which is where it was in the datadog parser

**Review by snopoke — CHANGES_REQUESTED** [review]
Can you also add the duration tag to line 26 so that we keep the prometheus metrics in line with the statsd ones: https://github.com/dimagi/commcare-logstash/pull/24/commits/677e754c513d7afcdfe306d18f2a4d2200290afc#diff-f400064efd695c6fcd63ee0efe26c411e164a0984cff45e6686b1624919081f5R26

**Comment by calellowitz** [comment]
> Can you also add the duration tag to line 26 so that we keep the prometheus metrics in line with the statsd ones: [677e754#diff-f400064efd695c6fcd63ee0efe26c411e164a0984cff45e6686b1624919081f5R26](https://github.com/dimagi/commcare-logstash/commit/677e754c513d7afcdfe306d18f2a4d2200290afc#diff-f400064efd695c6fcd63ee0efe26c411e164a0984cff45e6686b1624919081f5R26)

Done, thanks for catching that

**Review by snopoke — APPROVED** [review]
Not sure how important it is to fix the build - it's failing on downloading OpenJDK

**Comment by calellowitz** [comment]
Build continues to fail on the openjdk issue. Going to merge since it is metrics only and therefore low risk.

**Pull request merged**